instructionAPI/h/Expression.h

```diff
@@ -86,7 +87,10 @@ namespace Dyninst { namespace InstructionAPI {
     virtual bool bind(Expression* expr, const Result& value);
     virtual void apply(Visitor*) {}
 
-    virtual void getChildren(std::vector<Expression::Ptr>& children) const = 0;
+    DYNINST_DEPRECATED("Use getSubexpressions()")
+    virtual void getChildren(std::vector<Expression::Ptr>&) const {}
```

Comment:
You can make this non-virtual, implement here in the base and remove the implementation in the derived classes as their implementations are the same.

Reply:
Agreed. Updated.